core/control-plane/catalog-core/src/test/java/org/eclipse/edc/connector/catalog/DatasetResolverImplTest.java

```diff
@@ -111,8 +111,8 @@ void query_shouldReturnNoDataset_whenPolicyNotFound() {
 
     @Test
     void query_shouldReturnOneDataset_whenMultipleDefinitionsOnSameAsset() {
-        var policy1 = Policy.Builder.newInstance().build();
-        var policy2 = Policy.Builder.newInstance().build();
+        var policy1 = Policy.Builder.newInstance().assignee("assignee1").build();
```

Comment:
AFAIK the `assignee` and `assigner` fields is not used in EDC. could we remove them completely? Best done in a subsequent PR, because it will have a widespread impact

Reply:
ok, I was just looking for a way to distinguish the two policies, as `target` cannot be used because's overridden with the assetId, will look for another field